### PR TITLE
714 all case studies

### PR DESCRIPTION
**Pull request by nikeshashar**
Working on #714

### CODE REVIEW

**Comment by roidriscoll** [comment]
:shipit:

File: data/graduates.yml [hunk 1]
```diff
@@ -30,6 +30,66 @@
...
+  image: Mic.jpg
+  summary: IT Consultant to Developer at Pi-Top
+  testimonial: "Crazy Nerdy Rollercoaster!"
+  cohort_month: April
+  cohort_year: 2016
+  categories:
+   - career-changer
+  case_study:
+   - question: What were you doing before joining Makers Academy?
+     answer: I was an IT consultant at one of big four.
```
Comment (sjmog) [inline data/graduates.yml]: the big four?

File: data/graduates.yml [hunk 2]
```diff
@@ -30,6 +30,66 @@
...
+   - question: How did you come across Makers Academy and what made you decide to apply?
+     answer: I was looking for a coding course in London and I got a position at General Assembly. A friend of mine had done Makers that I read his blog which I followed. The attention to best practice is why I chose Makers Academy.
```
Comment (sjmog) [inline data/graduates.yml]: "A friend of mine had done Makers that I read his blog which I followed." I don't think this scans

File: data/graduates.yml [hunk 3]
```diff
@@ -30,6 +30,66 @@
...
+   - question: What did you find difficult on the course?
+     answer: The hardest thing was coding in pairs as it was a new skill in it self. Once you get used to it, it's one of the most enjoyable parts of the experience.
```
Comment (sjmog) [inline data/graduates.yml]: it self -> itself

File: data/graduates.yml [hunk 4]
```diff
@@ -30,6 +30,66 @@
...
+   - question: What was your end of cohort project?
+     answer: It was MarkPad - a Markdown text editor made from the ground up by our team.
+   - question: How has your life changed since learning to code?
+     answer: I do something that I love for living and can improve it everyday. Furthermore, I have infinite opportunities for the future which is a very significant change.
+   - question: What advice would you give to people thinking of changing career?
+     answer: Do your research and make sure to budget for it. If it's something you want to do then take a leap and you won't look back.
+   - question: What do you say to people who say £8000 is expensive?
+     answer: It's not even comparable to paying for a full Masters where you are having to support yourself for at least a year.
```
Comment (sjmog) [inline data/graduates.yml]: where you are having to => where you have to

File: data/graduates.yml [hunk 5]
```diff
@@ -30,6 +30,66 @@
...
+   - question: What was your biggest fear before joining MA and how do you feel about it now?
+     answer: I wasn't sure it would live up to my expectations or if I'd be a job ready Junior Developer but all I can say now that I enjoyed every minute of it and I'm still surprised by how much I learnt in a short while.
```
Comment (sjmog) [inline data/graduates.yml]: Run-on sentence; can we punctuate?

**Pull request merged**